### PR TITLE
Starting docs for image import.

### PR DESCRIPTION
**Pull request by crunk1**

### CODE REVIEW

**Comment by ericdand** [comment]
LGTM, except one thing: files in docs need fully-qualified URLs to link to non-docs pages, and vice versa. The former is essential: the links just won't work otherwise. The latter is just to help people find the GitHub.io page, rather than browsing our docs through the repo. I'll leave some in-line comments.

File: docs/image_import.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Image Import into Google Compute Engine (GCE)
+
+# Introduction
+We provide tooling and instructions for migrating your existing virtual
+machine images to GCE.
+
+Tools:
+* [Daisy](daisy.md)
+* [Import Precheck](../import_precheck/README.md)
```
Comment (ericdand) [inline docs/image_import.md]: Change this line to

    * [Import Precheck](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/import_precheck/

or

    * [Import Precheck](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/import_precheck/README.md

File: import_precheck/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Image Import Precheck Tool
+Precheck is run on your virtual machine before attempting to import it into
+Google Compute Engine (GCE). It attempts to identify compatibility issues that
+will either cause import to fail or will cause potentially unexpected behavior
+post-import. See our [image import documentation](../docs/image_import.md)
```
Comment (ericdand) [inline import_precheck/README.md]: Change this link to:

    [image import documentation](https://github.io/image_import.md)

Comment (crunk1) [reply import_precheck/README.md]: Done.

File: docs/image_import.md (hunk 1 above)
Comment (crunk1) [reply docs/image_import.md]: Done.

File: docs/image_import.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Image Import Process
+The image import process, also known as translation, imports virtual disks to
+GCE disks. Once a disk is imported, additional provisioning and configuration
+is usually needed to make the disk compatible with GCE. After importing,
+provisioning, and configuring, a GCE image is created from the GCE disk and the
+GCE disk is deleted.
```
Comment (chunjiali) [inline docs/image_import.md]: How this description relates to the import user guide doc?  Also talking about disk deletion in this high level introduction might cause confusion.

File: import_precheck/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Image Import Precheck Tool
+Precheck is run on your virtual machine before attempting to import it into
```
Comment (chunjiali) [inline import_precheck/README.md]: Precheck runs on ...

File: docs/image_import.md (hunk 3 above)
Comment (crunk1) [reply docs/image_import.md]: Modified the existing image import doc. Merged in this info.

File: docs/image-import.md [hunk 5]
```diff
@@ -225,11 +225,38 @@ $ daisy -var:source_image=projectsmy-awesome-projectglobal/images/my-server-impo
...
 
+Not every VM image will be importable to GCE. Some VMs will have issues after
+import. Below is a list of known compatibility requirements and issues:
+
+### Windows
+| Name | Severity | Description |
+|-|-|-|
+| OS Version | Required | We support the following OS versions: Windows Server 2008 R2, 2012 R2, or 2016; RHEL/CentOS/OEL 6 or 7; Debian 8 or 9; Ubuntu 14.04 or 16.04 |
+| OS Disk | Required | The disk containing the OS must be bootable and must be MBR. |
+| Multiple Disks | Warning  | Image import cannot handle multiple disk scenarios. Additional disks must be imported and attached separately. |
```
Comment (zmarano) [inline docs/image-import.md]: Image import cannot directly handle

File: docs/image-import.md [hunk 6]
```diff
@@ -225,11 +225,38 @@ $ daisy -var:source_image=projectsmy-awesome-projectglobal/images/my-server-impo
...
-*   Only virtual disks that use Grub as the default boot loader are supported.
 *   Networking: Import workflow sets the interface to DHCP. If that fails, or if there are other interfaces set with firewalls, special routing, VPN's, or other non-standard configurations, networking may fail and while the resulting instance may boot, you may not be able to access it.
 
+Not every VM image will be importable to GCE. Some VMs will have issues after
+import. Below is a list of known compatibility requirements and issues:
+
+### Windows
+| Name | Severity | Description |
+|-|-|-|
+| OS Version | Required | We support the following OS versions: Windows Server 2008 R2, 2012 R2, or 2016; RHEL/CentOS/OEL 6 or 7; Debian 8 or 9; Ubuntu 14.04 or 16.04 |
```
Comment (zmarano) [inline docs/image-import.md]: If this is Windows only mention only the Windows versions?

File: docs/image-import.md [hunk 7]
```diff
@@ -225,11 +225,38 @@ $ daisy -var:source_image=projectsmy-awesome-projectglobal/images/my-server-impo
...
+| OS Disk | Required | The disk containing the OS must be bootable and must be MBR. |
+| Multiple Disks | Warning  | Image import cannot handle multiple disk scenarios. Additional disks must be imported and attached separately. |
+| Powershell (Windows) | Warning | Warn if Powershell Version < 3. Powershell versions older than 3.0 can cause issues with GCE startup and shutdown scripts. |
+
+### Linux
+| Name | Severity | Description |
+|-|-|-|    
+| OS Version | Required | We support the following OS versions: RHEL/CentOS/OEL 6 or 7; Debian 8 or 9; Ubuntu 14.04 or 16.04. |
+| OS Disk | Required | The disk containing the OS must be bootable. The disk must be MBR and have GRUB installed. |
+| Multiple Disks | Warning  | Image import cannot handle multiple disk scenarios. Additional disks must be imported and attached separately. |
```
Comment (zmarano) [inline docs/image-import.md]: Image import cannot directly handle

File: docs/image-import.md [hunk 8]
```diff
@@ -225,11 +225,38 @@ $ daisy -var:source_image=projectsmy-awesome-projectglobal/images/my-server-impo
...
+| SSH | Warning | Warn if SSH is not running on port 22. GCE provides SSH clients via the Cloud Console and the gcloud CLI. These clients connect on port 22 and will not work if you have a different SSH configuration. |
+
+### Compatibility Precheck Tool
+Image import has a long runtime, can fail due to incompatibilities, and can
+cause unexpected behavior post-import. As such, you may find it useful to run
+our [precheck tool](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/import_precheck/)
+to check for the known issues listed above. See
+[Incompatibilities](#incompatibilities) above.
+
+There are binaries available for Windows and Linux.
```
Comment (zmarano) [inline docs/image-import.md]: Drop this here- its in the actual doc.

File: import_precheck/README.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# Image Import Precheck Tool
+Precheck runs on your virtual machine before attempting to import it into
+Google Compute Engine (GCE). It attempts to identify compatibility issues that
+will either cause import to fail or will cause potentially unexpected behavior
+post-import. See our [image import documentation](https://googlecloudplatform.github.io/compute-image-tools/image-import.md)
+for more information.
+
+Precheck must be run as root/admin.
```
Comment (zmarano) [inline import_precheck/README.md]: Precheck must be run as root or Administrator on the running system you want to import.

File: import_precheck/README.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+## Binaries
+Windows: https://storage.googleapis.com/compute-image-tools/release/windows/import_precheck_d03684b55aa87913fc7608694279a189379322ab.exe
```
Comment (zmarano) [inline import_precheck/README.md]: This should be setup the same way we do Daisy so there is a release and head build and we always point to the release build here.

File: docs/image-import.md (hunk 6 above)
Comment (crunk1) [reply docs/image-import.md]: Done.

File: docs/image-import.md (hunk 5 above)
Comment (crunk1) [reply docs/image-import.md]: Done.

File: docs/image-import.md (hunk 7 above)
Comment (crunk1) [reply docs/image-import.md]: Done.

File: docs/image-import.md (hunk 8 above)
Comment (crunk1) [reply docs/image-import.md]: Done.

File: import_precheck/README.md (hunk 9 above)
Comment (crunk1) [reply import_precheck/README.md]: Done.

File: import_precheck/README.md (hunk 10 above)
Comment (crunk1) [reply import_precheck/README.md]: Done.

**Pull request merged**